blackrock/templates/base.html

```diff
@@ -34,7 +40,7 @@
     {% block overlay %}{% endblock %}
 
 <div id="masthead_top">
-<a href="https://ccnmtl.columbia.edu/"><span id="ccnmtl"></span></a>
+<a href="https://www1.ccnmtl.columbia.edu/"><span id="ccnmtl"></span></a>
```

Comment:
This link https://www1.ccnmtl.columbia.edu/ isn't working for me. It looks like this should be http://ccnmtl.columbia.edu, because https://ccnmtl.columbia.edu forwards to http://www.columbia.edu/